psydac/api/tests/build_domain.py

```diff
@@ -180,7 +175,7 @@ def build_pretzel(domain_name='pretzel', r_min=None, r_max=None):
     mapping_14_2 = TransposedPolarMapping('M14_2',2, c1= r_min+h, c2= r_min+h, rmin = 0., rmax=1.)
     domain_14_2  = mapping_14_2(dom_log_14_2)
 
-    domain = union([
+    domains = ([
                     domain_1,
```

Comment:
As above.

Reply:
done in [d66a68e](https://github.com/pyccel/psydac/pull/309/commits/d66a68ed7d030e41dc86823c805ed685b78d906e)